src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/EditorsTextConstants_en.properties

```diff
@@ -91,7 +91,7 @@ onBreakpointSetPlumberfileWarning=Breakpoints not supported in Plumber API files
 errorSavingSetting=Error Saving Setting
 saveFile=Save File
 installTinyTeX=Installing TinyTeX
-updateDebugWarningBarMessage=Debug lines may not match because the file contains unsaved changes.
+updateDebugWarningBarMessage=Debug lines may not match because the editor contents have changed.

```

Comment:
If the debugger is active, the debug context will still remain out-of-sync even if you save the document, so I think this message is more accurate.